pyproject.toml

```diff
@@ -29,8 +29,9 @@ dependencies = [
   "SQLAlchemy~=1.4.42",
   "click>=7.1.2, <8.1.4",
   "databricks-sql-connector~=2.0",
-  "dbt-core==1.6.0b8",
-  "dbt-semantic-interfaces==0.1.0.dev8",
+  # TODO: Remove once there is a version of dbt-core that supports dbt-semantic-interfaces==0.1.0rc1
+  # "dbt-core==1.6.0b8",
+  "dbt-semantic-interfaces==0.1.0rc1",
```

Comment:
Just as an FYI, in core we're doing `dbt-semantic-interfaces~=0.1.0rc1`. I don't think pinning to `0.1.0rc1` here will be problematic though as we should be past breaking changes at this point

Reply:
Updated to use `~`.